### ISSUE TITLE
Displaying incomplete list of the torrent trackers from .torrent file

### ISSUE THREAD

**Issue by slrslr** [GerryFerdinandus/bittorrent-tracker-editor#44]
In some torrents, like [this one](http://leteckaposta.cz/558257948) the BTE 1.33.0.beta.6 on my Windows 10 does not not display all trackers. Just display first one, in this case leechers paradise. But when i open the file in text editor, i see more trackers.

https://github.com/GerryFerdinandus/bittorrent-tracker-editor/blob/master/test_torrent/Sintel.2010.720p.SURROUND.x264-VODO.torrent shows numerous trackers, OK

**Comment by GerryFerdinandus** [comment]
Thanks for the link. But I am unable to download the **Synové Velké Medvědice.torrent** Click on the link will not start the download etc.
Can you put the torrent as attachment in this tracker issue.

**Comment by slrslr** [comment]
@GerryFerdinandus link updated. If it wont download for you, use different browser.
Please edit your post to remove torrent name for privacy reasons.

**Comment by GerryFerdinandus** [comment]
Download now work.

The issue:
http://bittorrent.org/beps/bep_0012.html

Bittorrent tracker **only support this format:**
d['announce-list'] = [ [tracker1], [backup1], [backup2] ]
And each backup have one tracker item only.

But not this one.
d['announce-list'] = [[ tracker1, tracker2, tracker3 ]]
This is what your torrent is using

But also not this one.
d['announce-list'] = [ [ tracker1, tracker2 ], [backup1] ]

And there are many other combination possible with all variable different backup size in one torrent.

I do not support this because it is too complicated to implement for me.

**Comment by slrslr** [comment]
So in another words, if i create torrent, i should use empty lines between the tracker groups like [this](https://bin.disroot.org/?29b53d651186d391#39b6ubpHU5dumCaTfVxTWVfiBc93uWJn6mY7amuzdoWo)

```
a.trackerabc.com:123
b.trackerabc.com:123

trackerdef.eu/anounce

trackerghi.net:9034
```

I have seen that when i have not used empty lines between trackers, some torrent clients contact only one tracker.

torrent editor show only first tracker:

`d8:announce49:udp://tracker.leechers-paradise.org:6969/announce13:announce-listll49:udp://tracker.leechers-paradise.org:6969/announce42:udp://tracker.opentrackr.org:1337/announce29:udp://eddie4.nl:6969/announce34:http://mgtracker.org:6969/announce39:udp://inferno.demonoid.pw:3395/announce34:udp://p4p.arenabg.ch:1337/announce48:udp://tracker.internetwarriors.net:1337/announce43:udp://tracker.coppersurfer.tk:6969/announce34:http://torrentsmd.eu:8080/announce35:http://torrentsmd.com:8080/announceee7:comment61:...10:created by18:qBittorrent v4.1.013:`

torrent editor seem to show all trackers:

`d8:announce30:udp://9.rarbg.me:2710/announce13:announce-listll30:udp://9.rarbg.me:2710/announce31:udp://9.rarbg.com:2710/announce30:udp://9.rarbg.to:2710/announceel32:udp://explodie.org:6969/announceel39:udp://ipv4.tracker.harry.lu:80/announceel34:udp://p4p.arenabg.ch:1337/announce35:udp://p4p.arenabg.com:1337/announceel43:udp://tracker.coppersurfer.tk:6969/announceel38:udp://tracker.cyberia.is:6969/announceel48:udp://tracker.internetwarriors.net:1337/announceel49:udp://tracker.leechers-paradise.org:6969/announceel42:udp://tracker.opentrackr.org:1337/announceel41:udp://tracker.torrent.eu.org:451/announceee7:comment272:`

**Comment by GerryFerdinandus** [comment]
Yes, you must add empty lines between the tracker groups if you create a new torrent, normally the one that publish the tracker list already done so.
One example is to just copy and pasted from a validated/tested [tracker list](https://raw.githubusercontent.com/ngosang/trackerslist/master/trackers_best.txt)
